configs/branchmetrics.json

```diff
@@ -1,13 +1,11 @@
 {
-  "index_name": "branchmetrics",
-  "start_urls": [
-    {
-      "url": "https://docs.branch.io/(?P<lang>.*?)/",
-      "variables": {
-        "lang": ["", "ko", "ja", "zh"]
-      }
-    }
+ "index_name": "branchmetrics",
+ "start_urls": [
+   "https://docs.branch.io/"
   ],
+  algoliaOptions: {
```

Comment:
👋 @aopreaa ,

This parameter is here to filter [from your JS snippet](https://community.algolia.com/docsearch/config-file.html#using-custom-tags), it should not be included it here.